src/util/Numerical.js

```diff
@@ -255,7 +255,7 @@ var Numerical = new function() {
             if (D !== 0 && abs(D) < MACHINE_EPSILON) {
                 // If the geometric mean of the coefficients is small enough
                 var sc = (abs(a) + abs(b) + abs(c)) || MACHINE_EPSILON;
-                sc = pow(2, -Math.floor(Math.log(sc) / Math.log(2) + 0.5));
+                sc = pow(2, -Math.floor(Math.log(sc) * Math.LOG2E + 0.5));
```

Comment:
The result would be changed a little,
`sc = Math.exp(-Math.floor(Math.log(sc) + 0.5));`
is better?